programs/lz4cli.c

```diff
@@ -380,17 +378,20 @@ int main(int argc, const char** argv)
                 if (!strcmp(argument,  "--keep")) { LZ4IO_setRemoveSrcFile(0); continue; }   /* keep source file (default) */
                 if (!strcmp(argument,  "--rm")) { LZ4IO_setRemoveSrcFile(1); continue; }
                 if (longCommandWArg(&argument, "--fast")) {
-                        /* Parse optional window log */
+                        /* Parse optional acceleration factor */
                         if (*argument == '=') {
                             U32 fastLevel;
                             ++argument;
                             fastLevel = readU32FromChar(&argument);
                             if (fastLevel) {
                               cLevel = -(int)fastLevel;
                             }
+                            else {
```

Comment:
nit: We generally put the else on the same line as the previous closing brace `} else {`